cross_list_courses/static/cross_list_courses/js/controllers/ListController.js

```diff
@@ -60,13 +60,15 @@
                         next: '',
                         previous: '',
                     },
+                    processing: '<img src="' + window.globals.STATIC_URL + 'images/ajax-loader-large.gif" class="loading"/> Loading...',
```

Comment:
Any reason we went with this instead of the fontawesome one we're using elsewhere? https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/37f7596e569967b265727d583fb2b0491bab4f4f/course_info/templates/course_info/partials/details.html#L11